File: TeamUp.API/Models/Place.cs
using System.Collections.Generic;

namespace TeamUp.API.Models
{
    public class Place
    {
        public int Id {get; set;}
        public string Name {get; set;}
        public string Country {get; set;}
        public string City {get; set;}
        public string Address {get; set;}
        public string HowToBook {get; set;}
        public string Info {get; set;}
        public string Latitude {get;set;}
        public string Longitude {get;set;}
        public string FieldType {get;set;}
        public string FieldSize {get;set;}
        public bool Public {get;set;}
        public ICollection<PlacesPhoto> PlacesPhoto  {get; set;}
    
    }
}

File: TeamUp-SPA/src/app/_models/place.ts
import { PlacesPhoto } from './placesPhoto';

export interface Place {
  id: number;
  name: string;
  country: string;
  city: string;
  address: string;
  howToBook: string;
  info: string;
  latitude: number;
  longitude: number;
  fieldType: string;
  fieldSize: string;
  public: boolean;
  photos?: PlacesPhoto[];
}
